org.ksnip.ksnip.yaml

```diff
@@ -6,6 +6,7 @@ command: ksnip
 rename-icon: ksnip
 finish-args:
   - --socket=x11
```

Comment:
```suggestion
  - --socket=fallback-x11
```

since there is wayland

Reply:
I've committed your suggestion.
Does this mean that, if I am running Wayland (so it is available), I wouldn't be able to make the application use X11 unless I open up regular `--socket=x11`?

Reply:
yes